collections/_posts/2023-09-15-SAM_for_Medical.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: review
+title: "Segment Any Medical Image"
+tags: deep-learning medical image segment-anything
+author: "Robin Trombetta"
+cite:
+    authors: "ZJun Ma, Yuting He, Feifei Li, Lin Han, Chenyu You, Bo Wang"
+    title:   "Segment Anything in Medical Images"
+pdf: "https://arxiv.org/pdf/2304.12306.pdf"
+---
+
+# Note
+
+This is a combined review of the four following papers. The aim is to give an idea about how Segment Anything (SAM)[^1] has been used and adapted to medical images since its released in April 2023.
+
+* J. Ma, Y. He, F. Li, L. Han, C. You, B. Wang, **Segment Anything in medical imaging**, April 2023, 
+* K. Zhang, D. Liu, **Customized Segment Anything Model for Medical Image Segmentation**, April 2023
+* J. Wu, R. Fu, Y. Zhang, H. Fang, Y. Liu, Z.Wang, Y. Xu, Y. Jin, **Medical SAM Adapter: Adapting Segment Anything Model for Medical Image Segmentation**, April 2023
+* X. Lin, Y. Xiang, L. Zhang, X. Yang, Z. Yan, L. Yu, **SAMUS: Adapting Segment Anything Model for Clinically-Friendly and Generalizable Ultrasound Image Segmentation**, September 2023
```

Comment:
I suggest adding hyperlinks directly to the papers (e.g. arXiv versions).

Reply:
Thanks for the suggestions, I added the links